content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -0,0 +1,182 @@
+---
+reviewers:
+- thockin
+- dwinship
+min-kubernetes-server-version: v1.29
+title: Extend Service IP Ranges
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+This document shares how to extend the existing Service IP range assigned to a cluster.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## API
+
+Kubernetes clusters that are created with the `MultiCIDRServiceAllocator`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will create a new ServiceCIDR object with the well-known name `kubernetes` from the kube-apiserver `--service-cluster-ip-range` flag values.
+
+```sh
+kubectl get servicecidr
+```
+```
+NAME         CIDRS          AGE
+kubernetes   10.96.0.0/28   17d
+```
+
+The well-known `kubernetes` Service, that exposes the kube-apiserver endpoint to the Pods, uses as ClusterIP the first IP from the default ServiceCIDR.
+
+```sh
+kubectl get service kubernetes
+```
+```
+NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   17d
+```
+
+The default Service, in this case, uses the ClusterIP 10.96.0.1, that has the corresponding IPAddress object.
+
+```sh
+kubectl get ipaddress 10.96.0.1
+```
+```
+NAME        PARENTREF
+10.96.0.1   services/default/kubernetes
+```
+
+The ServiceCIDRs are protected with finalizers, to avoid leaving Service ClusterIPs orphans,
```

Comment:
```suggestion
The ServiceCIDRs are protected with {{<glossary_tooltip text="finalizers" term_id="finalizer">}}, to avoid leaving Service ClusterIPs orphans;
```